### ISSUE TITLE
`func config git set` command fails when `build.pvcSize` field is empty.

### ISSUE THREAD

**Issue by jrangelramos** [knative/func#1687]
I'm trying `func config git set` (using a built from main) on a function project created by the latest func released [v1.9.4](https://github.com/knative/func/releases/tag/knative-v1.9.4) 

I got the below error
```
$ grep pvc func.yaml 
$ func config git set
? The URL to Git repository with the function source code: https://github.com/jrangelramos/hello-node-func
? The Git branch or tag we are targeting: main
? A subpath within the repository: 
? Do you want to configure webhook trigger? No
 ✅ Pipeline template is created in ".tekton/pipeline.yaml"
 ✅ Pipeline template is created in ".tekton/pipeline-run.yaml"

E0424 15:09:12.377539 1230987 v2.go:105] io: read/write on closed pipe
Error: failed to run generate pipeline: PVC size value cannot be parsed due to: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
```

It happens bacause `build.pvcSize` field is empty on `func.yaml`. On a func project created from a build from main, this field is default to [256Mi](https://github.com/knative/func/pull/1598/files#diff-6580ab49d6ac9f67db49c596955cf575901492bbe31560c2c69d0b1dfdeba8a8R34)

I would suggest a better message to end user, by instructing developer `pvcSize` field is missing/empty on `build` structure.
--or--
Perhaps use the default value of 256Mi when PVC is empty.

**Comment by zroubalik** [comment]
@jrangelramos I cannot reproduce against `main`. Could you please double check? Thanks!

**Comment by jrangelramos** [comment]
fixed by #1771
